### ISSUE TITLE
Add darker/lighter to context responses (i.e. alerts)

### ISSUE THREAD

**Issue by terracoda** [phetsims/molarity#160]
There was feedback from consultant that we should be describing that the solution gets darker and light. In addition there is a learning goal noted in the teaching tips that students should be able to associate "concentration" and the shade of the solution.

@Matthew-phet and I discussed the need to add "darker"/"lighter" parameters gracefully into the alerts we already have. Over to @Matthew-phet.

These new parameters will describe the solution getting darker when it gets more concentrated and lighter when the solution gets less concentrated.

**Comment by Matthew-phet** [comment]
@terracoda is this kind of what we were thinking?
While changing solute amount provide progress:
- {{More}} solute, {{more}} concentrated. Color darkens.
- {{Less}} solute, {{less}} concentrated. Color lightens.

While changing solution amount provide progress:
- {{More}} solution, {{less}} concentrated. Color lightens.
- {{Less}} solution,{{more}} concentrated. Color darkens.

**Comment by Matthew-phet** [comment]
While changing solute amount provide progress:

{{More}} solute, {{more}} concentrated. Solution darkens.
{{Less}} solute, {{less}} concentrated. Solution lightens.
While changing solution amount provide progress:

{{More}} solution, {{less}} concentrated. Solution lightens.
{{Less}} solution,{{more}} concentrated. Solution darkens.

**Comment by emily-phet** [comment]
@Matthew-phet and @terracoda 
What about (for the examples above) just stating "Darker" or "Lighter" instead of "Solution darkens" or "solution lightens". If that doesn't seem good, how about "Solution darker" and "Solution lighter"?

**Comment by terracoda** [comment]
I am adding my comments from slack (Oct 22) just so we have them.
> I like the color parameter at the end of the alert as I think it is less important. "Solution darker"/ "Solution lighter" could also work. 

> Do you know or can you confirm that the solution does not get darker after saturation point. In which case, you won't want that parameter after saturation. 

> Also, can you check to see if tagging this on also works (doesn't sound awkward) when Solution Values is checked.

> "Solution lighter/darker" is one syllable longer, so consider what's more relevant to the entire experience, the word, "color" which we don't mention very often, or the word "solution" which is around a lot in the State Descriptions, and mostly implied in the alerts.

> You need to verify all places where the new "strings for the alerts" may or may not appear before assigning over to Taylor. We are close to the final design and we don't want to introduce any bugs

**Comment by emily-phet** [comment]
Just a note on solution colors after saturation - it should stay the same. No more solute is going "into" solution to create a color change.

**Comment by terracoda** [comment]
@Matthew-phet, I think I prefer an adjective to a verb in this case. Are amenable to swapping out lightens/darkens with lighter/darker?

**Comment by terracoda** [comment]
@Matthew-phet, @twant, and @emily-phet, this issue is very related to #154, and I don't think the examples provided here cover all affected strings.

@twant, @Matthew-phet, @emily-phet could you all please see the the examples I have worked out in the design document for Changing Solute.

I am going to work through Changing Solution Volume, too. It should be very similar, but mapped in the opposite way.

**Comment by terracoda** [comment]
Please see [Context Responses](https://docs.google.com/document/d/1Z_5biEOBk9W4lVPXxGt7fyg-V6_CwTA4vMZ0mVMTJsI/edit#heading=h.ijildz9ngmpl) section in the design doc

**Comment by twant** [comment]
@terracoda and @Matthew-phet just pushed some updates with the lighter/darker descriptions. Let me know what you think!

**Comment by terracoda** [comment]
@twant, this is looking good. I'm working through examples with Solution Values checked in the design doc.

**Comment by terracoda** [comment]
@twant, all the phrases for darker/lighter in both modes are now updated in the design doc.
See [Context Responses section](https://docs.google.com/document/d/1Z_5biEOBk9W4lVPXxGt7fyg-V6_CwTA4vMZ0mVMTJsI/edit#heading=h.ijildz9ngmpl)

I also added a table to try and summarize everything, but I think I have only captures Solute Amount changes. See [Table: Values describing Shade](https://docs.google.com/document/d/1Z_5biEOBk9W4lVPXxGt7fyg-V6_CwTA4vMZ0mVMTJsI/edit#heading=h.w9dol87dx4ss)

Apologies for taking so long. I was trying to make it as clear as possible for you, so it would be easy to implement. Everything that has been implemented is not highlighted and is not marked in red.

Needed changes are highlighted and marked in red. **Note** that bracketing is not consistent, so please do what makes sense.= for the code.

**Comment by terracoda** [comment]
@twant, apologies, I forgot to change the labels and assign to you on Friday. I don't see any commits since Friday to this issue, so assigning to you now.

**Comment by twant** [comment]
@terracoda @Matthew-phet implemented what's in the design doc for these responses so far. Still working on modifying the "no solute" alerts.

**Comment by twant** [comment]
@terracoda @Matthew-phet I just implemented the change in no solute alerts from the design doc. Not sure if that change belongs with this issue, but if it does, I'm assigning over to you both now for review of that change, as well as review of all of the lighter/darker context alerts.

**Comment by terracoda** [comment]
@twant, the responses (alerts) for no solute are working great!

Ad I noted over in #154, the Table containing the shade descriptions is all updated. It now includes example responses for both Solute Amount changes and Solution Volume changes, and it has also includes progress responses with and without region changes. 

Everything seems to be working great with Solution Values NOT checked.

With Solution Values checked there are still some issues which are all highlighted in the table in the design doc. 

The parameters "lighter", "darker", and "darkest" are all showing up at the right time. It's just the surrounding strings and phrases when Solution Values is checked that need work.

I am comfortable with you closing this issue. I think all other needed changes to the strings/phrases will happen over in #154 .

Please re-open if I have closed prematurely.
